packages/api-explorer/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "6.15.1",
   "main": "dist/index.js",
   "dependencies": {
-    "@readme/http-status-codes": "^7.0.0",
+    "@readme/http-status-codes": "^7.1.0",
```

Comment:
This dep update gave us support for accepting `default` as a valid status code. See https://github.com/readmeio/http-status-codes/pull/7